docs/controllers/extendedjob.md

```diff
@@ -20,63 +24,96 @@ There are two different kinds of `ExtendedJob`:
 - **one-offs**: automatically runs once after it's created
 - **errands**: needs to be run manually by a user
 
-![flow-extended-job](quarks_deployment_flow-ExtendedJob.png)
+## ExtendedJob Component
 
-## Features
+The **ExtendedJob** component is a categorization of a set of controllers, under the same group. Inside the **ExtendedJob** component, we have a set of 2 controllers together with 2 separate reconciliation loops.
```

Comment:
See second comment or so. Have this awkward phrasing in all the docs. Will not highlight the other places anymore.